src/Util/Search.php

```diff
@@ -678,4 +684,21 @@ private function getCollectionEndDateQuery(array $collectionDates): Query\Range
 
         return $collectionEndDateRange;
     }
+
+    /**
+     * Get the UDI query.
+     *
+     * @param string $queryTerm Query term that needs to be searched upon.
+     *
+     * @return Query\Match
+     */
+    private function getUdiQuery(string $queryTerm): Query\Match
+    {
```

Comment:
perhaps a regex in here to determine if the term is an UDI, and if it is, use the matchPhrase. On discovery, we also allowed the use of periods in place of the colon then substituted the colon back in.

Reply:
I replaced this with the existing query with MatchPhrase, the way we do it  on data-discovery. I get only 1 result, even though that UDI is mentioned in another dataset's abstract (this is how it works on Data-discovery as well)

Reply:
I changed it to Matchphrase, and i figured that we were nulling the query term in data-discovery. Now it works the way we want. Its ready for test